cocos/2d/CCSprite.cpp

```diff
@@ -614,6 +614,17 @@ void Sprite::draw(Renderer *renderer, const Mat4 &transform, uint32_t flags)
     }
 }
 
+TrianglesCommand::Triangles Sprite::getRenderedTriangles() const
+{
+    static unsigned short indices[6] = {0, 1, 2, 3, 2, 1};
```

Comment:
I've noticed that using a static in a method doesn't work on Android. 
For example:

```cpp
for (int i = 0; i < 10; i++)
{
    static int a = 0;
    a++;
}
```
Result will always be 1.

I might be wrong, but i think i had problem with that.